src/System-Hashing/MD5NonPrimitive.class.st

```diff
@@ -25,7 +25,7 @@ Class {
 
 { #category : #'class initialization' }
 MD5NonPrimitive class >> initialize [
-	"MD5 initialize"
+	"MD5NonPrimitive initialize"
 	"Obscure fact: those magic hex numbers that are hard to type in correctly are
```

Comment:
This isn't related to the other edit, but was just a comment I stumbled across that seems wrong.